integration/compareDissecCentralized.spec.js

```diff
@@ -0,0 +1,101 @@
+const { execSync } = require('child_process')
+const {
+  default: CozyClient,
+  createClientInteractive,
+  Q
+} = require('cozy-client')
+
+const { populateCentralized } = require('../scripts/populateCentralized')
+const { dissecLearning } = require('./helpers/dissecLearning')
+const { localLearning } = require('./helpers/localLearning')
+const { BANK_DOCTYPE } = require('../src/doctypes/bank')
+const { JOBS_DOCTYPE } = require('../src/doctypes/jobs')
+
+describe('Compares the performance of a centralized learning vs the DISSEC one', () => {
+  const defaultTimeout = 150000
+
+  const uri = 'http://test1.localhost:8080'
+  let client
+  let token
+  let operations = []
+  let validationSet = []
+  let categories = []
+  let cutoffDate
+
+  beforeAll(async () => {
+    await populateCentralized()
+
+    token = execSync(
+      `cozy-stack instances token-app ${uri.replace('http://', '')} dissecozy`
+    )
+      .toString()
+      .replace('\n', '')
+
+    // Helper
+    const getCategory = doc => {
+      return doc.manualCategoryId || doc.localCategoryId || doc.cozyCategoryId
+    }
+
+    // Connect to the instance
+    client = await (async () => {
+      const schema = {
+        operations: {
+          doctype: BANK_DOCTYPE,
+          attributes: {},
+          relationships: {}
+        }
+      }
+      if (token) {
+        return new CozyClient({
+          uri,
+          schema,
+          token: token
+        })
+      } else {
+        return await createClientInteractive({
+          scope: [BANK_DOCTYPE, JOBS_DOCTYPE],
+          uri,
+          schema,
+          oauth: {
+            softwareID: 'io.cozy.client.cli'
+          }
+        })
+      }
+    })()
```

Comment:
As this is used elsewheren this could be in a helper, so you just have to call `client = await getClient(doctypes, {token})`